src/ACEtk/block/ProbabilityTable/src/generateXSS.hpp

```diff
@@ -0,0 +1,35 @@
+static std::vector< double >
+generateXSS( std::vector< double >&& probabilities,
+             std::vector< double >&& total,
+             std::vector< double >&& elastic,
+             std::vector< double >&& fission,
+             std::vector< double >&& capture,
+             std::vector< double >&& heating ) {
+
+  std::vector< double > xss;
+  std::size_t bins = probabilities.size();
+  xss.reserve( 6 * probabilities.size() );
+
+  if ( ( total.size() != bins ) || ( elastic.size() != bins ) ||
+       ( fission.size() != bins ) || ( capture.size() != bins ) ||
+       ( heating.size() != bins ) ) {
+
+    Log::error( "The size of data in the probability table are not the same" );
+    Log::info( "Expected size = {}", bins );
+    Log::info( "Found probabilities ize = {}", probabilities.size() );
+    Log::info( "Found total ize = {}", total.size() );
```

Comment:
All `size` were written as `ize` here and the next few lines.

Reply:
I've done a refactor later that removed this file from ProbabilityTable